psgdpr.php

```diff
@@ -492,7 +492,7 @@ public function hookAdditionalCustomerFormFields($params)
                 break;
         }
 
-        if ($active == false) {
+        if ($active === false) {
```

Comment:
This strict comparison may lead to errors, as Configuration::get(...) usually returns a string.

```suggestion
        if ($active === false) {
```